docs/troubleshooting/02_rpcerror.md

```diff
@@ -0,0 +1,72 @@
+When a call to the chain_api is performed and fails, it will result in an RPCError object being generated to inform why the transaction failed.
```

Comment:
maybe:

When a call to the chain_api is performed and fails, it will result in an RPCError object being generated to inform why the transaction failed.

to =>

When a call to the chain_api is performed and fails, it will result in an RPCError object being generated which contains information on why the transaction failed.